src/app/content/utils/urlUtils.spec.ts

```diff
@@ -173,11 +178,6 @@ describe('toRelativeUrl', () => {
 
   it('ignores trailing slashes on the target', () => {
```

Comment:
delete these tests about ignoring the trailing slash, that is no longer desired behavior

Reply:
I've removed them